Tiltfile

```diff
@@ -27,3 +27,8 @@ k8s_yaml('./test/ingress.yaml')
 
 # Metallb
 k8s_yaml('./test/metallb.yaml')
+
+# Gateway API
```

Comment:
Oh, I just realized you might want to add a `k8s_kind()` directive in here for the Gateway objects.
I'm still new to using Tilt, so maybe it isn't necessary, but I did end up adding one for VirtualServers:
`k8s_kind('VirtualServer', api_version='k8s.nginx.org/v1')`

Reply:
ha, interesting, I haven't seen this before. I did see Tilt complaining about some CRDs but didn't pay attention to it. Thanks, I'll look it up.